src/f2c/derf_.c

```diff
@@ -4,10 +4,14 @@ extern "C" {
 #endif
 
 #ifdef KR_headers
+#ifndef HAVE_ERF
```

Comment:
I don't understand this.

This file appears to **assume** that the `erf()` function is available, and define `derf_` (a fortran function) in terms of it. 

Why did you condition it on an `#ifndef HAVE_ERF`?

It is my impression that this version of f2c wouldn't work at all without having `erf()`. If we don't have `erf()`, configuration should just fail.